delta-app/src/main/java/io/cdap/delta/app/DeltaWorker.java

```diff
@@ -127,12 +127,16 @@ public void initialize(WorkerContext context) throws Exception {
     OffsetAndSequence offsetAndSequence = deltaContext.loadOffset();
     offset = offsetAndSequence.getOffset();
 
+    Set<DDLOperation> ddlBlacklist = new HashSet<>(config.getDdlBlacklist());
+    // targets will not behavior properly if they don't get create table events
```

Comment:
nit - behave

Reply:
ah yes. my fingers sometimes auto-complete to the wrong word.